cartographer_ros/cartographer_ros/node.cc

```diff
@@ -261,6 +261,16 @@ void Node::PublishLocalTrajectoryData(const ::ros::TimerEvent& timer_event) {
         node_options_.use_pose_extrapolator
             ? ToRos(now)
             : ToRos(trajectory_data.local_slam_data->time);
+
+    // Suppress publishing if we already published a transform at this time.
+    // Due to 2020-07 changes to geometry2, tf buffer will issue warnings for
+    // repeated transforms with the same timestamp
```

Comment:
We use full sentences in comments, so please end it with a point character as mentioned by @ojura.